.github/workflows/expired.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '* * */1 * *'
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          exempt-issue-milestones: 'future,alpha,beta,release'
+          exempt-pr-milestones: 'bugfix,improvement'
+          exempt-all-pr-assignees: true
+          stale-issue-message: 'This issue is stale because it has been open 60 days with no activity. Remove stale label or comment or this will be closed in 15 days.'
+          stale-pr-message: 'This PR is stale because it has been open 60 days with no activity. Remove stale label or comment or this will be closed in 15 days.'
+          close-issue-message: 'This issue was closed because it has been stalled for 15 days with no activity.'
+          close-pr-message: 'This PR was closed because it has been stalled for 15 days with no activity.'
+          days-before-issue-stale: 60
```

Comment:
While the stale bot is helpful in practice, I think we should be more generous with the timing, i.e. 60 -> 120, 15 -> 30.

Reply:
The time has been increased